src/nrepl/jack-in.ts

```diff
@@ -41,6 +41,10 @@ function cancelJackInTask() {
 }
 
 async function executeJackInTask(projectType: projectTypes.ProjectType, projectTypeSelection: any, executable: string, args: any, cljTypes: string[], outputChannel: vscode.OutputChannel, connectSequence: ReplConnectSequence) {
+    // If the connection sequence specifies a project root dir, override the existing project dir
+    const projectRootDir = connectSequence?.menuSelections?.projectRootDir;
+    if(projectRootDir) state.setProjectRoot(projectRootDir);
```

Comment:
Minor detail. The code base might not show it, but I am trying to move in the direction where statement bodies of `if`s and such are enclosed in curly brackets. See https://github.com/BetterThanTomorrow/calva/wiki/Coding-Style#curlies

Reply:
Works for me, should have read the style guide more thoroughly.